### DISCUSSION TITLE
Allgemeingültigkeit

### DISCUSSION THREAD

**Discussion by durek1337** [BixConcept/tutoring-frontend#83] [Ideas]
Da zur Zeit eig. eh nur Deutsche an diesem Projekt arbeiten, schreibe ich mal in Deutsch:

Das Projekt ist gut gelungen und es ist ein großartiger Beitrag für mehr Bildung. Nachhilfe ist etwas Tolles. Es nützt den Lernenden, den Lehrenden und dient hier der Schule selbst auch als weiteres Bildungsangebot. Da es ja sogar OpenSource ist, würde ich vorschlagen, dass es früher oder später von allen festgeschriebenen GymHaan-Sachen weggeht und vielleicht in Installationsskript im Browser zu bedienen wäre, wenn die Datenbankstruktur noch nicht vorhanden ist oder so. Da könnte man dann den Namen und die E-Mail-Domain der Schule festlegen (sowie die Daten zum SMTP-Server verändern) und ein Logo hochladen. Generell sollte die Installation so niederschwellig wie möglich sein. NodeJS ist da leider nicht die beste Wahl, da die wenigsten Lehrkräfte (welche sich letztlich um sowas kümmern werden) Ahnung davon haben und webserver mit NodeJS-Unterstützung seltener schon gemietet sind, als welche mit php-Unterstützung. Dann würden andere Schulen das vielleicht eher nutzen.

Mein Vorschlag: Macht euch **bei Gelegenheit** Gedanken über die Änderbarkeit und Übertragbarkeit des Projekts, sodass es hoffentlich irgendwann von anderen Schulen adaptiert wird.

**Comment by 3nt3** [comment]
Früh gelesen (/s), aber ist natürlich eigentlich sinnvoll :+1:
